build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/BaseInternalPluginBuildPlugin.java

```diff
@@ -81,29 +81,6 @@ public void doCall() {
         if (isModule == false || isXPackModule) {
             addNoticeGeneration(project, extension);
         }
-        project.afterEvaluate(p -> {
```

Comment:
Actually this was only used by internal plugins that 

a) still rely on test cluster plugin, 
b) extend other plugins
c) do not use default distribution

Therefore instead of changing the internal plugin api to always provide a project mapping, we just removed it and fixed that ONE project that relies on this snippet.